task.proto

```diff
@@ -27,11 +27,17 @@ message Authentication {
     string access_token = 1;
 }
 
+enum TaskDifficulty {
+    easy = 0;
+    medium = 1;
+    hard = 2;
+}
```

Comment:
gRPC enums should be either in PascalCase or SCREAMING_SNAKE_CASE. The first value should be reserved for `UNSPECIFIED`.

See https://buf.build/docs/best-practices/style-guide#enums

Reply:
Done